### DISCUSSION TITLE
Enrolling Endpoints into fleet with custom URL

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#10929] [2.4] [answered]
I want to start by saying just how good of an upgrade over 2.3 2.4 has been shaping up to be. I mean its night and day, I don't ever want to go back. I eagerly await GA.

Speaking to this question though, I have been trying to ship logs over the internet from endpoints. I saw that you added custom fleet servers URLs and changing this setting, I can see it was added as fleet URL and a Logstash output. 

(This could absolutely just be me going at this wrong) But when I attempt to run the installer provided on a different network, it fails to find the fleet server. In the txt file generated, it shows to have only attempted connections to the default URLs, not having tried the custom one. 

Is there a way to update the installer to include the custom URL for enrollment purposes? Is it as simple as reordering the URLs in the fleet settings in Kibana? Or is this a case of me being totally mistaken, the ladder being extremely probable.

**Comment by dougburks** [comment, marked as answer]
This should be resolved in the upcoming 2.4 RC2 release!

**Reply by jaypfe** [reply]
Awesome!
